webgasdospela/MI2part1.core

```diff
@@ -70,7 +70,8 @@ start() >> {hatch := Closed;
 { player.loc = LaundryShip } ?
     { hatch.sticked != True, hatch = Open } ?
         walkto(path) >> {player.loc := WoodtickLeft; hatch := Closed} & { "You hear the hatch close as you go back to the several-ships place" }
-    walkto(path) >> {player.loc := WoodtickLeft} & { "You are among several ships" }
+    { hatch.sticked != True, hatch != Open } ?
+        walkto(path) >> {player.loc := WoodtickLeft} & { "You are among several ships" }
```

Comment:
I don't think there's anything to update. I fixed the issue that was here previously, By removing a rule.